db/migrate/20220319150610_create_organizations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrganizations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :organizations do |t|
+      t.string :name, NULL: false
+      t.string :address
+      t.integer :phone
+      t.string :email, NULL: false
+      t.text :welcomeText, NULL: false
```

Comment:
It's a convention in Rails to use lower snake case for compound database field names.
https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e